index.html

```diff
@@ -20,6 +20,10 @@ <h1>Example Recommendations Component</h1>
     <script>
       window.cruRecommendationsComponent.render(
         document.querySelector('#cru-recommendations-component'),
+        {
+          uri:
+            'https://www.cru.org/us/en/about/partners/a-surprise-in-one-of-baltimores-toughest-neighborhoods.html',
```

Comment:
I assume this is a default that we override by passing in a `uri`?

Reply:
Or, I see code there to pull it from the location.

Reply:
Ya you can either pass in a uri or it will fall back to the `window.location.href`. I added this so I could make my examples work.